src/NHibernate.Test/Async/TransformTests/AliasToBeanResultTransformerFixture.cs

```diff
@@ -257,6 +257,8 @@ public void ToPropertiesInsensitivelyDuplicated_WithoutAnyProjectionsAsync()
 		[Test]
 		public async Task SerializationAsync()
 		{
+			Assume.That(typeof(System.Type).IsSerializable);
```

Comment:
Since it is put at many places, refactoring this into a common helper may help detect later all tests which needs to be re-enabled if #1425 gets merged. (If the code above is always identical, it could also be found again by a simple search, but still a common helper will be slightly cleaner.)

Reply:
Done